src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppElasticBeanstalk.recipe

```diff
@@ -329,6 +329,126 @@
+                    "Type": "String",
+                    "DefaultValue": "Time",
+                    "AllowedValues": [
+                        "Time",
+                        "Health",
+                        "Immutable"
+                    ],
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "ElasticBeanstalkRollingUpdates.RollingUpdatesEnabled",
+                            "Value": true
+                        }
+                    ]
+                },
+                {
+                    "Id": "MaxBatchSize",
+                    "Name": "Max Batch Size",
+                    "Description": "The number of instances included in each batch of the rolling update.",
+                    "Type": "Int",
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "ElasticBeanstalkRollingUpdates.RollingUpdatesEnabled",
+                            "Value": true
+                        }
+                    ],
+                    "Validators": [
+                        {
+                            "ValidatorType": "RangeOrNull",
+                            "Configuration": {
+                                "Min": 1,
+                                "Max": 10000
+                            }
+                        }
+                    ]
+                },
+                {
+                    "Id": "MinInstancesInService",
+                    "Name": "Min Instances In Service",
+                    "Description": "The minimum number of instances that must be in service within the Auto Scaling group while other instances are terminated.",
+                    "Type": "Int",
+                    "AdvancedSetting": true,
+                    "Updatable": true,
+                    "DependsOn": [
+                        {
+                            "Id": "ElasticBeanstalkRollingUpdates.RollingUpdatesEnabled",
+                            "Value": true
+                        }
+                    ],
+                    "Validators": [
+                        {
+                            "ValidatorType": "RangeOrNull",
+                            "Configuration": {
+                                "Min": 0,
+                                "Max": 9999
+                            }
+                        }
+                    ]
+                },
+                {
+                    "Id": "PauseTime",
+                    "Name": "Pause Time",
+                    "Description": "The amount of time (in seconds, minutes, or hours) the Elastic Beanstalk service waits after it completed updates to one batch of instances and before it continues on to the next batch.",
```

Comment:
Give an example of the format like you do for `Timeout`.

Reply:
We should have a validation RegEx for the setting.

Reply:
Done